mod_reforged/hooks/skills/racial/alp_racial.nut

```diff
@@ -1,4 +1,16 @@
 ::mods_hookExactClass("skills/racial/alp_racial", function(o) {
+	o.onAdded = function()
```

Comment:
I just realized, this function doesn't exist in the original file (in vanilla). So you need to do `<-`. Can you please check for the other files too and update the code accordingly?